sow/search.md

```diff
@@ -0,0 +1,405 @@
+training of new personnel (contractor or State) during transition
+period, appropriate close-out of outstanding technical and related
+work.
+
+Final report shall include list of sprint tasks completed,
+documentation, and link to code repository. Should
+the Contractor be terminated prior to the end of the period of
+performance, the Contractor shall transfer all project materials to
+the COR within two weeks of the COR’s request.
+
+#### 5.6 Deliverables
+
+##### Table 1 List of Deliverables
+**Deliverable** | **Due Date** | **Description**
+--- | --- | ---
+Code & Status Reports | 1 business day after each sprint | Demonstration of progress throughout each sprint.
+Code Repository of Product | End of call order | Version-controlled Open Source repository of code that comprises prototype.
+Research | A research plan shall be delivered during the first sprint. Research-related records shall be delivered at the end of the second sprint and every applicable sprint. thereafter | A summary of research conducted and results found. If applicable, next steps or recommendations based on research.
+Design Deliverables | End of every applicable sprint | Mock ups and/or design files if applicable, or design changes reflected in the Development Prototype.
+Development View | End of second sprint and every sprint thereafter | In-progress development view, accessible on the web via staging server / development server.
+Transition Plan | To be worked on no later than the second to last sprint of the initial Period of Performance | Work with the Alaska product team to make sure there is a clear plan for handing off the code repository.
+
+The contractor shall submit all deliverables to the Alaska product team.
+
+##### 5.6.1 Delivery Instructions [Defer to Nick/Simon]
+
+Code deliverables shall be submitted via the Github repository. A copy
+of any document deliverables shall be submitted to the Alaska product team.
+
+##### 5.6.2 Inspection and Acceptance of Services [Defer to Nick/Simon]
+
+All periodic reports and task deliverables shall be inspected, tested
+(where applicable), reviewed, and accepted by the Alaska product team within 5
+days of the conclusion of each sprint.
+
+Only the Alaska product team has the authority to
+inspect, accept, or reject all deliverables. 
+
+##### 5.6.3 System Documentation
+
+The Contractor shall consult with the Alaska product team to determine what is
+appropriate, effective, and essential for system documentation. The
+State requires, at a minimum, that the contractor will generate
+comprehensive and complete documentation, both within the code itself,
+within the source code version control system (e.g., through proper
+use of descriptive commit messages, issue tracking, pull requests,
+etc.), and as appropriate, in separate documentation, provide
+artifacts, and create new user stories based on each sprint.
+
+##### 5.6.4. Quality Assurance [defer to Nick]
+
+[Specify AK Quality Assurance requirements]
```

Comment:
Contractor shall comply with standards set forth in the quality assurance plan (QAP) (updated link to the QAP in Github)